libs/common/src/sql-db-entities/facility-utilisation-data/facility-utilisation-data.entity.ts

```diff
@@ -42,4 +48,11 @@ export class FacilityUtilisationDataEntity extends AuditableBaseEntity {
     data.updateLastUpdatedBy(requestSource);
     return data;
   }
+
+  public updateWithFixedFeeUtilisationAndReportPeriod({ fixedFee, utilisation, reportPeriod, requestSource }: UpdateWithFixedFeeAndReportPeriodParams): void {
```

Comment:
Bit of a nitpick on the naming here:

Utilisation and fixed fee are separated above (`createWithoutUtilisationAndFixedFee`), but joined here (`updateWithFixedFeeUtilisationAndReportPeriod`, the `FixedFeeUtilisation` substring) and utilisation is then excluded in the type `UpdateWithFixedFeeAndReportPeriodParams`.

Could be good to make these all the same, eg:
- createWithoutUtilisationAndFixedFee -> createWithoutFixedFeeAndUtilisation
- updateWithFixedFeeUtilisationAndReportPeriod -> updateWithFixedFeeAndUtilisationAndReportPeriod
- UpdateWithFixedFeeAndReportPeriodParams -> UpdateWithFixedFeeAndUtilisationAndReportPeriodParams

Reply:
This naming convention is based on the naming convention we tend to follow in test files ie. `withABAndC` should read as `with A, B and C`. However, the naming convention we have tended to follow in other entities has been to name the `update` method with something context specific. In this case, I think a good choice for the name will be `updateWithCurrentReportPeriodDetails`

Reply:
That's great, thank you for following up